src/sentry/static/sentry/app/utils/withSentryAppInstallations.jsx

```diff
@@ -1,78 +0,0 @@
-import React from 'react';
-
-import {Client} from 'app/api';
-import SentryTypes from 'app/sentryTypes';
-import getDisplayName from 'app/utils/getDisplayName';
-import SentryAppInstallationStore from 'app/stores/sentryAppInstallationsStore';
-
-const withSentryAppInstallations = WrappedComponent => {
```

Comment:
Why's this removed?

Reply:
The logic was moved over to the `fetchSentryAppInstallations()`. Seemed easier to explicitly call that than wrap a bunch of things in the HOC. We can change it back if that's the pattern we want.